securing_apps/topics/oidc/java/logout.adoc

```diff
@@ -4,9 +4,10 @@
 You can log out of a web application in multiple ways.
 For Jakarta EE servlet containers, you can call `HttpServletRequest.logout()`. For other browser applications, you can redirect the browser to
 `\http://auth-server{kc_realms_path}/{realm-name}/protocol/openid-connect/logout`, which logs the user out if that user has an SSO session with his browser. The actual logout is done once
-the user confirms the logout. You can optionally include parameters such as `id_token_hint`, `post_logout_redirect_uri` and others as described in the
+the user confirms the logout. You can optionally include parameters such as `id_token_hint`, `post_logout_redirect_uri`, `client_id` and others as described in the
 https://openid.net/specs/openid-connect-rpinitiated-1_0.html[OpenID Connect RP-Initiated Logout]. As a result, that logout does not need to be explicitly confirmed
-by the user.
+by the user if `id_token_hint` parameter is included. After logout, user will be automatically redirected to the specified `post_logout_redirect_uri` as long as it is provided.
+Note that it is needed to include either `client_id` or `id_token_hint` parameter in case that `post_logout_redirect_uri` is included.
```

Comment:
```suggestion
Note that you need to include either the `client_id` or `id_token_hint` parameter in case that `post_logout_redirect_uri` is included.
```